kfs-core/src/main/resources/edu/arizona/kfs/fp/document/datadictionary/ProcurementCardDefaultMaintenanceDocument.xml

```diff
@@ -29,112 +12,140 @@
-        <bean parent="MaintainableFieldDefinition" p:name="cardHolderWorkPhoneNumber" p:required="false" />            
-        <bean parent="MaintainableFieldDefinition" p:name="cardLimit" p:required="false" />  
-        <bean parent="MaintainableFieldDefinition" p:name="cardCycleAmountLimit" p:required="false" /> 
-        <bean parent="MaintainableFieldDefinition" p:name="cardCycleVolumeLimit" p:required="false" />  
-        <bean parent="MaintainableFieldDefinition" p:name="cardStatusCode" p:required="false" />  
-        <bean parent="MaintainableFieldDefinition" p:name="cardNoteText" p:required="false" />
+        <bean parent="MaintainableFieldDefinition" p:name="cardHolderSystemId" p:unconditionallyReadOnly="true" p:required="true"  />
+        <bean parent="MaintainableFieldDefinition" p:name="cardHolderLine1Address" p:unconditionallyReadOnly="true" p:required="false"  />
+        <bean parent="MaintainableFieldDefinition" p:name="cardHolderLine2Address" p:unconditionallyReadOnly="true" p:required="false"  />
+        <bean parent="MaintainableFieldDefinition" p:name="cardHolderCityName" p:unconditionallyReadOnly="true" p:required="false" />
+        <bean parent="MaintainableFieldDefinition" p:name="cardHolderStateCode" p:unconditionallyReadOnly="true" p:required="false" />
+        <bean parent="MaintainableFieldDefinition" p:name="cardHolderZipCode"  p:unconditionallyReadOnly="true" p:required="false" />
+        <bean parent="MaintainableFieldDefinition" p:name="cardHolderWorkPhoneNumber" p:unconditionallyReadOnly="true" p:required="false" />
       </list>
     </property>
   </bean>
-  
-  <bean id="ProcurementCardDefaultMaintenanceDocument-EditProcurementCardAccounting" parent="ProcurementCardDefaultMaintenanceDocument-EditProcurementCardAccounting-parentBean" />
-
-  <bean id="ProcurementCardDefaultMaintenanceDocument-EditProcurementCardAccounting-parentBean" abstract="true" parent="MaintainableSectionDefinition">
-    <property name="id" value="KFS-FP_ProcurementCardDefault-EditProcurementCardAccounting" />
+  <bean id="ProcurementCardDefaultMaintenanceDocument-EditPCardSettings" parent="ProcurementCardDefaultMaintenanceDocument-EditPCardSettings-parentBean" />
+  <bean id="ProcurementCardDefaultMaintenanceDocument-EditPCardSettings-parentBean" abstract="true" parent="MaintainableSectionDefinition">
+    <property name="id" value="KFS-FP_ProcurementCardDefault-EditPCardSettings" />
+    <property name="title" value="Edit Default Procurement Card Settings" />
+    <property name="maintainableItems" >
+      <list>
+        <bean parent="MaintainableFieldDefinition" p:name="cardLimit" p:unconditionallyReadOnly="true" p:required="false"  />
+        <bean parent="MaintainableFieldDefinition" p:name="cardCycleAmountLimit" p:unconditionallyReadOnly="true" p:required="false"  />
+        <bean parent="MaintainableFieldDefinition" p:name="cardCycleVolLimit" p:unconditionallyReadOnly="true" p:required="false"  />
```

Comment:
this looks correct now. See previous reviews where we might have missed the slight name change for this field.